.ci/functional_tests_utils.sh

```diff
@@ -98,5 +98,6 @@ function archiveArtifacts() {
   mkdir -p ./rhche/${JOB_NAME}/${BUILD_NUMBER}/surefire-reports
   cp ./logs/*.log ./rhche/${JOB_NAME}/${BUILD_NUMBER}/
   cp -R ./logs/artifacts/screenshots/ ./rhche/${JOB_NAME}/${BUILD_NUMBER}/
+	cp -R ./logs/artifacts/failsafe-reports/ ./rhche/${JOB_NAME}/${BUILD_NUMBER}/
```

Comment:
@rhopp indent looks incorrect, + what is `/failsafe-reports` ?

Reply:
yep.. Indentation is wrong. I noticed that... I'll fix that before merging.
failsafe-reports is directory, where testng results are kept. But as I look at it, we still need some tweaking there: http://artifacts.ci.centos.org/devtools/rhche/devtools-rh-che-rh-che-prcheck-dev.rdu2c.fabric8.io/1254/failsafe-reports/